src/main.rs

```diff
@@ -258,6 +260,13 @@ fn main() {
             }
         }
 
+        "clean" => {
+            match fs::remove_dir_all(&config.dest) {
```

Comment:
`cobalt build` doesn't create `dest` directory:
```
$ cobalt build
[info]   Using config file ./.cobalt.yml
[info]   Building from . into build
[error]  IO error: No such file or directory (os error 2)
[error]  Build not successful
```

I think it meanse `cobalt clean` shouldn't remove it.